### PR TITLE
Display a message when adding/updating/removing users

### PR DESCRIPTION
**Pull request by huwshimi**
Done:
- Added global messages actions/reducers.
- Added a Notification component.
- Updated users form/list to show a message when users are created, updated or deleted. Fixes: https://github.com/canonical-web-and-design/maas-squad/issues/1410. Fixes: https://github.com/canonical-web-and-design/maas-ui/issues/108.

QA:
- Add, update or delete a user.
- You should get a message.

### CODE REVIEW

**Comment by lilyvidenova** [comment]
@huwshimi Can we just put a full stop at the end of each message?

File: ui/src/app/base/components/Section/Section.js [hunk 1]
```diff
@@ -23,6 +42,7 @@ const Section = ({ children, sidebar, title }) => {
           </Col>
         )}
         <Col size={sidebar ? 10 : 12} className="section__content">
+          {generateMessages(messages, dispatch)}
```
Comment (Caleb-Ellis) [inline ui/src/app/base/components/Section/Section.js]: This could probably be a separate component - NotificationsList or something.

**Comment by huwshimi** [comment]
@lilyvidenova Thanks, I've added the full stops.

File: ui/src/app/base/components/Section/Section.js (hunk 1 above)
Comment (huwshimi) [reply ui/src/app/base/components/Section/Section.js]: Good idea. Done.

**Pull request merged**